apps/codeforafrica/src/components/NavListItem/index.js

```diff
@@ -0,0 +1,35 @@
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+const NavLinkItemRoot = styled("li")(
+  ({ theme: { breakpoints, typography } }) => ({
+    marginBottom: `${typography.pxToRem(20)}`,
+    listStyle: "none",
+    [breakpoints.up("lg")]: {
+      marginBottom: `${typography.pxToRem(10)}`,
+      listStyle: "none",
+    },
+  })
```

Comment:
```suggestion
  ({ theme: { breakpoints, typography } }) => ({
    listStyle: "none",
  })
```

`margin` is dependent on `direction` (or where it's being used) and not screen size. So either leave it out and set it on say `FooterNavList` or in the `NavList` component, set `margin` of child li (`& > li`) depending on `direction`

Reply:
This hasn't been addressed.